src/commands/compile/internal/spec/compileTestFile.spec.ts

```diff
@@ -30,22 +30,44 @@ describe('compileTestFile', () => {
   let target: Target
   let sasjsPath: string
   let testBody: string
+  let buildPath: string
   const testFileName = 'random.test.sas'
 
   beforeAll(async () => {
-    testBody = await readFile(path.join(__dirname, 'testFiles', testFileName))
-
     process.logger = new Logger(LogLevel.Off)
 
     appName = `cli-tests-compile-test-file-${generateTimestamp()}`
+
+    buildPath = path.join(__dirname, appName, 'sasjsbuild')
+
     target = await createTestGlobalTarget(appName, '/Public/app')
```

Comment:
Maybe the target that comes from `createTestGlobalTarget` can include the config that you're adding below?

Reply:
good sugestion!
done